L/libgifextra/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Giflibextra"
```

Comment:
This name _**must**_ match the name of the directory where the file is in

Reply:
> current name in build_tarballs.jl will work right?

I'm not sure if I understand this; do you mean the case differences "Giflibextra" vs "GifLibExtra"? I believe BB doesn't differentiate this but it's better to be consitent.

---

When GifLibExtra is ready (with valid licenses), you can tag a release v0.0.1. Then I believe it should properly pass Yggdrasil build, assuming the build script works.